etc/scripts/install-on-minikube.sh

```diff
@@ -31,7 +31,19 @@ minikube config set cpus ${CPUS}
 minikube config set disk-size ${DISK_SIZE}
 
 # Start minikube
-minikube start -p knative --extra-config=apiserver.enable-admission-plugins="LimitRanger,NamespaceExists,NamespaceLifecycle,ResourceQuota,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook"
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
-"$DIR/install.sh" -q
+if [-z "$1"]; then
+  minikube start -p knative --extra-config=apiserver.enable-admission-plugins="LimitRanger,NamespaceExists,NamespaceLifecycle,ResourceQuota,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook"
+  exit_status=$?
+else
+  minikube start -p knative --vm-driver $1 --extra-config=apiserver.enable-admission-plugins="LimitRanger,NamespaceExists,NamespaceLifecycle,ResourceQuota,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook"
+  exit_status=$?
+fi
+
+if [ $exit_status -eq 0 ]; then
```

Comment:
Good call checking whether minikube started successfully! I'd prefer to just check `$?` instead of introducing `exit_status` though.

Reply:
Yup, `exit_status` is unnecessary.